pkgs/desktops/gnome-3/misc/geary/default.nix

```diff
@@ -1,62 +1,56 @@
-{ stdenv, fetchurl, intltool, pkgconfig, gtk3, vala_0_40, enchant
-, wrapGAppsHook, gdk_pixbuf, cmake, ninja, desktop-file-utils
-, libnotify, libcanberra-gtk3, libsecret, gmime, isocodes
-, gobject-introspection, libpthreadstubs, sqlite, gcr
-, gnome3, librsvg, gnome-doc-utils, webkitgtk, fetchpatch }:
+{ stdenv, fetchurl, pkgconfig, gtk3, vala, enchant, wrapGAppsHook, gdk_pixbuf, meson, ninja
+, desktop-file-utils, gnome-online-accounts, gsettings-desktop-schemas, adwaita-icon-theme
+, libnotify, libcanberra-gtk3, libsecret, gmime, isocodes, libxml2, gettext, gobject-introspection
+, libpthreadstubs, sqlite, gcr, json-glib, itstool, libgee, gnome3, librsvg, webkitgtk, python3
+, xvfb_run, dbus, shared-mime-info, libunwind }:
 
-let
-  pname = "geary";
-  version = "0.12.4";
-in
 stdenv.mkDerivation rec {
-  name = "${pname}-${version}";
+  pname = "geary";
+  version = "0.13.1";
 
   src = fetchurl {
-    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${name}.tar.xz";
-    sha256 = "01ykhkjfkprvh9kp4rzrl6xs2pqibiw44ckvqsn5cs3xy2rlq8mm";
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "0h9pf2mdskq7mylib1m9hw86nwfmdzyngjl7ywangqipm1k5svjx";
   };
 
-  patches = [
-    # Fix build with webkitgtk-2.22
-    (fetchpatch {
-      url = https://gitlab.gnome.org/GNOME/geary/commit/5d0f711426d76f878cf9b71f7e8f785199c7cde1.patch;
-      sha256 = "1yifng5lfsc6wp7irmi8gjdcfig1cr0chf7rdv3asrk567nmwrsi";
-    })
-    (fetchpatch {
-      url = https://gitlab.gnome.org/GNOME/geary/commit/0d966950a2cba888873cd3a7f4f42bb7a017dc6d.patch;
-      sha256 = "1y6v4fnik4w3paj9nl0yqs54998sx1zr9w3940d579p6dsa8f3fg";
-    })
-    (fetchpatch {
-      url = https://gitlab.gnome.org/GNOME/geary/commit/e091f24b00ec421e1aadd5e360d1550e658ad5ef.patch;
-      sha256 = "0d5hc4h9c1hnn2sk18nkpmzdvwm3h746n2zj8n22ax9rj6lxl38l";
-    })
-    # Fix build with vala 0.40.12
-    # See: https://gitlab.gnome.org/GNOME/vala/blob/0.40.12/NEWS#L22
-    (fetchpatch {
-      url = "https://gitlab.gnome.org/GNOME/geary/commit/088cb2c0aa35ad4b54ea5a0a2edaf0ff96c64b45.patch";
-      sha256 = "0cnjmbd3snm8ggmprqa32f7i3w86gs3ylab9p5ffj921dcpvvlb2";
-    })
+  nativeBuildInputs = [
+    desktop-file-utils gettext gobject-introspection itstool
```

Comment:
Do we still need `gobject-introspection` for vala apps with https://github.com/NixOS/nixpkgs/pull/54324?

Reply:
This is a rare case https://gitlab.gnome.org/GNOME/geary/blob/master/meson.build#L63

We probably should do a treewide cleanup for that.

Reply:
Turns out it is actually just a relic: https://gitlab.gnome.org/GNOME/geary/merge_requests/138